src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -338,7 +338,7 @@ private void BuildFallbackBlock(TagHelperContent builder)
 
                 // Build the <meta /> tag that's used to test for the presence of the stylesheet
                 builder
-                    .AppendHtml("<meta name=\"x-stylesheet-fallback-test\" class=\"")
+                    .AppendHtml("<meta property=\"x-stylesheet-fallback-test\" content class=\"")
```

Comment:
that's 'cos you're validating against XHTML 1.1. We don't care about that. Validate it with `<!doctype html>` in order to invoke the HTML5 rules.